src/components/board/Card.jsx

```diff
@@ -1,7 +1,52 @@
 import React from "react";
+import Image from "next/image";
+import Judge from "../../../public/images/judge.png";
+import { GrMail } from "react-icons/gr";
 
-const Card = () => {
-  return <div>Card</div>;
+const Card = (props) => {
+  const { name, title, year, major, pronouns, linkedIn, email } = props;
+
+  return (
+    <div className="cardContainer font-poppins m-4 bg-white w-[265px] h-[400px] rounded-3xl shadow flex flex-col items-center justify-center">
```

Comment:
Don't use pixels are large elements like this. Use this instead: https://tailwindcss.com/docs/width